File: C++/largest of three numbers.cpp
#include<iostream>
using namespace std;

int main(){
     cout<<"Find the Largest of the three numbers."<<endl<<endl;
     int no_1,no_2,no_3;
     cout<<"Enter Number 1 = ";
     cin>>no_1;
     cout<<"Enter Number 2 = ";
     cin>>no_2;
     cout<<"Enter Number 3 = ";
     cin>>no_3;
  
     if(no_1>=no_2 && no_1>=no_3){
          cout<<"Number 1 is the Largest";
     }else if(no_2>no_1 && no_2>no_3){
          cout<<"Number 2 is the Largest";
     }else if(no_3>no_1 && no_3>no_2){
          cout<<"Number 3 is the Largest";
     }else{
          cout<<"Error";
     }
  
     return 0;
}

File: Python./Largest of three numbers.py
no_1 = int(input("Enter the No-1 : "))
no_2 = int(input("Enter the NO-2 : "))
no_3 = int(input("Enter the NO-3 : "))

if no_1 > no_2 and no_1 > no_3:
    print("No-1 is the Largest.")
elif no_2 > no_1 and no_2 > no_3:
    print("No-2 is the Largest.")
elif no_3 > no_2 and no_3 > no_1:
    print("No-3 is the Largest.")
elif no_1 == no_2 == no_3 :
    print("All are equal")
else :
    print("Error")
